docs/branching/branch-and-commit.md

```diff
@@ -12,7 +12,7 @@ example:
 
 # Commit naming strategy:
 
-Always include **prefix** DOC2-**issue-id** in your commit message, so that valid commit message can be only:
+Always include prefix **DAT-issue-id** in your commit message, so that valid commit message can be only:
```

Comment:
in jira we have DOC. So, maybe for avoid inconvenience we will continue use DOC instead of DAT?

Reply:
@max402 so DAT or DOC?